dashboard/app/models/ability.rb

```diff
@@ -121,6 +121,15 @@ def initialize(user)
         can :read, :pd_workshop_summary_report
         can :read, :pd_teacher_attendance_report
       end
+
+      if user.permission?(UserPermission::WORKSHOP_ADMIN)
+        can :manage, Pd::Workshop
+        can :manage, Pd::Workshop
```

Comment:
The tests already exist, for admin: https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/controllers/pd/workshop_material_orders_controller_test.rb#L89-L90

They just need to be adjusted for workshop_admin.

Should we block admin access to these pages now like we do for projects, or leave it as-is where it will sort-of partially work?